src/System.Management.Automation/engine/regex.cs

```diff
@@ -57,12 +57,12 @@ public sealed class WildcardPattern
         // The size is less than MaxShortPath = 260.
         private const int StackAllocThreshold = 256;
 
+        // chars that are considered special in a wildcard pattern
+        private const string SpecialChars = "*?[]`";
```

Comment:
@daxian-dbw This constant field is only used once, perhaps it could be made a constant local in the `Init()'` method?

Reply:
It doesn't matter for this PR. :-)